### PR TITLE
Update volcanoplot

### PR DESCRIPTION
**Pull request by paulzierep**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/volcanoplot/volcanoplot.xml [hunk 1]
```diff
@@ -283,6 +307,45 @@ sessionInfo()
...
+        <param name="lfc_col" value="2" />
+        <param name="label_col" value="1" />
+        <param name="shape_col" value="5" /> <!-- Assuming the shape is in column 5 -->
+        <param name="lfc_thresh" value="0" />
+        <param name="label_select" value="file"/>
+        <param name="label_file" ftype="tabular" value="labels.tab" />
+        <param name="shape_or_facet" value="facet" /> <!-- Testing the facet option -->
+        <output name="plot">
+            <assert_contents>
+                <has_size value="5007" delta="1000" />
```
Comment (bgruening) [inline tools/volcanoplot/volcanoplot.xml:325]: We do have better imaging asserts nowadays, if you want to give it a try.

**Review by bgruening — APPROVED** [review]

File: tools/volcanoplot/volcanoplot.xml [hunk 2]
```diff
@@ -248,9 +270,11 @@ sessionInfo()
                 </assert_contents>
             </output>
         </test>
+
         <test expect_num_outputs="1">
             <!-- Ensure input labels and plot options work -->
             <param name="input" ftype="tabular" value="input.tab"/>
+            <param name="fdr_col" value="5" />
```
Comment (bgruening) [inline tools/volcanoplot/volcanoplot.xml:277]: fdr_col twice?

**Comment by bgruening** [comment]
ping @paulzierep

File: tools/volcanoplot/volcanoplot.xml (hunk 1 above)
Comment (paulzierep) [reply tools/volcanoplot/volcanoplot.xml:325]: you mean `compare="image_diff"`, or are there other undocumented diff options, I think not needed here

Comment (bgruening) [reply tools/volcanoplot/volcanoplot.xml:324]: image_diff but there are other image assertions: https://docs.galaxyproject.org/en/latest/dev/schema.html#has-image-center-of-mass

**Review by bgruening — APPROVED** [review]

**Comment by pcm32** [comment]
@bgruening the macros here are not being used in the version, and then this was uploaded to the toolshed without a version change, see https://github.com/galaxyproject/tools-iuc/pull/6510/files#diff-fa1711e49e2c00dd46c9d86d082e9091e5468442a270582b30e3c92cdad4557dR1

**Comment by pcm32** [comment]
This explains #6620

**Comment by bgruening** [comment]
Do you see the problem in this PR?

**Comment by bgruening** [comment]
Ah I see. I can release a new version with bump, will this solve your problem?

**Comment by pcm32** [comment]
I would prefer if the 0.0.6 version is reinstated, I have a bunch of workflows pinned to that. Also this is breaking an IWC workflow.

**Comment by pcm32** [comment]
And then the newer changes are pushed in a different version

**Pull request merged**